utils/port_utils.py

```
@@ -0,0 +1,272 @@
+                    break
+                chunks.append(data)
+            except socket.timeout:
+                break
+
+        raw = b"".join(chunks).decode("utf-8", errors="replace")
+        # 分离响应头与响应体
+        if "\r\n\r\n" not in raw:
+            return None
+        _, body = raw.split("\r\n\r\n", 1)
+
+        # 处理 chunked 传输编码（常见单块场景）
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)